references/detection/presets.py

```diff
@@ -2,12 +2,22 @@
 
 
 class DetectionPresetTrain:
-    def __init__(self, hflip_prob=0.5):
-        trans = [T.ToTensor()]
-        if hflip_prob > 0:
-            trans.append(T.RandomHorizontalFlip(hflip_prob))
-
-        self.transforms = T.Compose(trans)
+    def __init__(self, data_augmentation, hflip_prob=0.5, mean=(123., 117., 104.)):
```

Comment:
I guess that the `mean=(123., 117., 104.)` here should be changed to `[0.48235, 0.45882, 0.40784]` too, I noticed that you have rescaled it in the previous commit.

Reply:
Yes, these values are before the scaling, so they should be as is.